src/polkadot/fellows.ts

```diff
@@ -91,6 +91,18 @@ export class PolkadotFellows implements TeamApi {
     }
   }
 
+  /** Returns all the fellows with their rankings */
+  async listFellows(): Promise<[string, number][]> {
+    this.logger.info("Fetching all fellows with their ranks");
+
+    if (this.fellowsCache.size < 1) {
+      this.logger.debug("Cache not found. Fetching fellows.");
+      this.fellowsCache = await this.fetchAllFellows();
```

Comment:
to the previous comment of fetching fellows - this function should either end up with positive amount of fellows (more than one) or throw exception, as it's not expected to not return anything, then perhaps there's some underlying problem with our api or rpc or url etc
just as sanity check

Reply:
We do have [this check](https://github.com/paritytech/review-bot/blob/45730962868c3f45a224746cd34ea439a0c281f2/src/polkadot/fellows.ts#L121) for a method that is always called before that new method is invoked.